docs/usage/webhook/webhook.rst

```diff
@@ -0,0 +1,158 @@
+Redeploying on receving github webhook
+======================================
+
+We'll move  ``ffmpeg`` to a github repo,and set up the webhook dataflow such that whenever we push
+to the default branch,the new version is pulled and a docker container is build and run with this
+new version.
+
+Deploying on docker container
+-----------------------------
+
+We'll first setup a docker container for running ffmpeg http service.A ``Dockerfile`` is
+already generated in ffmpeg.We need to modify it to include ``ffmpeg``.
+
+**Dockerfile**
+
+.. literalinclude:: /../examples/ffmpeg/Dockerfile
+
+.. note::
+    The run command in the comment section of the Dockerfile will be used to execute
+    the container after receving webhooks,so make sure you change it to your usecase.
+
+For this tutorial we will change it to
+
+.. code-block:: console
+
+    docker run --rm -d -v ~/Videos:/usr/src/app/data -ti -p 8080:8080 aghinsa/ffmpeg \
+        -mc-config deploy -insecure -log debug
+
+you should replace ``aghinsa/ffmpeg`` with your ``USERNAME/REPONAME``,in both run and build commands,in cli
+and in comment section of ``Dockerfile``.
+
+The additional flags are:
+
+    - -d : start the container in detached mode
+    - -v : map the directory ``~/Videos`` in host to ``/usr/src/app/data`` in container
```

Comment:
Let's follow the same format as https://intel.github.io/dffml/master/quickstart/model.html#quickstart-command-line-flags-explained for consistency.

```suggestion
- ``-v ~/Videos:/usr/src/app/data``

  - Map the directory ``~/Videos`` in host to ``/usr/src/app/data`` in container
```